src/User.js

```diff
@@ -3,14 +3,30 @@ class User {
     this.id = userData.id;
     this.name = userData.name;
     this.address = userData.address;
+    this.email = userData.email
     this.strideLength = userData.strideLength;
     this.dailyStepGoal = userData.dailyStepGoal;
-    this.friends = userData.friends;
+    this.friends = userData.friends
+    this.friendsNames = []
   }
 
   getFirstName() {
     return this.name.split(' ')[0];
   }
+
+  getUserFriendNames() {
+    let userIds = userData.filter(user => {
+      return user.id
+    })
+    userIds.forEach((i) => {
+      if(this.friends.includes(i.id)) {
+        this.friendsNames.push(i.name)
+      }
+    })
+  }
```

Comment:
We've already got a method on the UserRepository class to get info about a user.  Maybe when you're populating the friends list in the DOM you could do something like this on line 23 of scripts.js:

`friendsList.innerHTML = user.friends.map(friendID => userRepo.getUserData(friendID).name`

Reply:
Ill change this

Reply:
Forgot a ) at end but works great there